README.md

```diff
@@ -8,3 +8,165 @@ You can choose to use either the:
 - [React COGS SDK](./packages/javascript/README.md)
 
 Learn more in the [COGS technical documentation](https://docs.cogs.show).
+
+## Quick start
+
+Add the COGS JavaScript SDK to your project:
+
+```bash
+npm install --save @clockworkdog/cogs-client
+# OR
+yarn add @clockworkdog/cogs-client
+```
+
+or if you are using React:
+
+```bash
+npm install --save @clockworkdog/cogs-client-react
+# OR
+yarn add @clockworkdog/cogs-client-react
+```
+
+## COGS Media Master custom content
+
+### What is a Media Master custom content?
+
+COGS Media Masters can display a browser window with custom HTML, Javascript and CSS, using the COGS SDK.
+
+Some other features you can add to your custom content include:
+
+- Play audio from your COGS project
+- Display or react to COGS text hints
+- Show the COGS show timer
+- Connect to MIDI or WebSerial devices
+
+### Contents of Media Master custom content
+
+A folder in the `client_content` directory of a COGS Project, containing:
+
+- A [`cogs-plugin-manifest.js` manifest file](https://clockwork-dog.github.io/cogs-sdk/javascript/interfaces/CogsPluginManifestJson.html)
+- An HTML page with the content of your plugin, usually `index.html`
+- Some Javascript, loaded by the HTML page, that uses the [COGS SDK](https://github.com/clockwork-dog/cogs-sdk) to communicate with COGS.
+
+## COGS Plugins
+
+### What is a Plugin?
+
+COGS Plugins can be loaded into COGS to integrate with other systems or add specialised features to COGS.
+
+### Contents of a COGS Plugin
+
+A COGS plugin is a folder in the `plugins` directory of a COGS Project. The plugin directory name is the ID of the plugin (e.g. `dog.clockwork.http`) and contains:
+
+- A [`cogs-plugin-manifest.js` manifest file](https://clockwork-dog.github.io/cogs-sdk/javascript/interfaces/CogsPluginManifestJson.html)
+- An HTML page with the content of your plugin, usually `index.html`
+- Some Javascript, loaded by the HTML page, that uses the [COGS SDK](https://github.com/clockwork-dog/cogs-sdk) to communicate with COGS.
+
+### What is a Plugin?
```

Comment:
This seems to be a repeat of above. I presume you meant to only include one of them?

Reply:
I meant to repeat it actually. I think it was confusing that we referred to both custom content and plugins as "plugins". This (I think) makes it clear you can use the SDK for both with the same structure, even through their uses are different.

Reply:
Oh I see, the plugin bit was repeated again! Fixed.